docs/runbooks/CDINotReady.md

```diff
@@ -1,32 +1,48 @@
 # CDINotReady
+<!--Edited by davozeni, 17.10.2022-->
 
 ## Meaning
 
-The CDINotReady alert is indicative of a CDI installation being in a degraded state;  
+The CDINotReady alert indicates that a containerized data importer (CDI) installation is in a degraded state:
```

Comment:
This PR is almost ready. The only change I request is that you change the wording of the first sentence so that it matches the other runbooks for consistency:

"This alert fires when a containerized data importer (CDI) installation is in a degraded state:"